endpoints/put_test.go

```diff
@@ -677,50 +947,58 @@ func TestEmptyPutRequests(t *testing.T) {
 			desc:    "All empty body. ",
 			reqBody: "{}",
 			expected: testOutput{
-				jsonResponse:      `{"responses":\[\]}`,
-				statusCode:        http.StatusOK,
-				metricsBadRequest: 0,
-				metricsDuration:   1.00,
+				jsonResponse:          `{"responses":\[\]}`,
+				statusCode:            http.StatusOK,
+				durationMetricsLogged: true,
 			},
 		},
 		{
 			desc:    "Empty puts arrray",
 			reqBody: "{\"puts\":[]}",
 			expected: testOutput{
-				jsonResponse:      `{"responses":\[\]}`,
-				statusCode:        http.StatusOK,
-				metricsBadRequest: 0,
-				metricsDuration:   1.00,
+				jsonResponse:          `{"responses":\[\]}`,
+				statusCode:            http.StatusOK,
+				durationMetricsLogged: true,
 			},
 		},
 	}
 
-	for i, test := range testCases {
+	for i, tc := range testCases {
 		// Set up server
 		backend := backends.NewMemoryBackend()
-		m := metricstest.CreateMockMetrics()
+		mockMetrics := metricstest.CreateMockMetrics()
+		m := &metrics.Metrics{
+			MetricEngines: []metrics.CacheMetrics{
+				&mockMetrics,
+			},
+		}
 		router := httprouter.New()
 		router.POST("/cache", NewPutHandler(backend, m, 10, true))
 		rr := httptest.NewRecorder()
 
 		// Create request everytime
-		request, err := http.NewRequest("POST", "/cache", strings.NewReader(test.reqBody))
+		request, err := http.NewRequest("POST", "/cache", strings.NewReader(tc.reqBody))
 		assert.NoError(t, err, "[%d] Failed to create a POST request: %v", i, err)
 
 		// Run
 		router.ServeHTTP(rr, request)
 		//assert.Equal(t, http.StatusOK, rr.Code, "[%d] ServeHTTP(rr, request) failed = %v \n", i, rr.Result())
```

Comment:
Can this commented out code be deleted? I know this wasn't an addition to this PR, but the clean up could be nice.

Reply:
Forgot to clean up a little bit. Removed.